guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p><strong>Note:</strong> In the absence of <code>arcpy</code>, the <a href="https://pypi.org/project/Fiona/" target="_blank">Fiona</a> package must be present in your current conda environment in order to read a featureclass. &nbsp;To check if Fiona is present, you can run the following in a cell: </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>